doc/source/cluster/vms/user-guides/faq.rst

```diff
@@ -0,0 +1,56 @@
+.. _cluster-FAQ:
+
+FAQ
+===
+
+What is the difference between ``--node-ip-address`` and ``--address``?
+-----------------------------------------------------------------------
+
+When starting a head node, ``ray start --node-ip-address xx.xx.xx.xx --port
+nnnn`` is used to specify the address of the head node. When starting a worker
+node, ``ray start --address xx.xx.xx.xx:nnnn`` is used to specify the head node
+this worker should connect to.
+
+What does a worker node failure to connect look like?
+-----------------------------------------------------
+
+If the worker node cannot connect to the head node, you should see this error:
+::
```

Comment:
<img width="802" alt="Screenshot 2023-06-02 at 4 58 10 PM" src="https://github.com/ray-project/ray/assets/5459654/6e58ecb6-523b-4ddd-a56c-acecc1e6d84d">
It's using code syntax highlighting by default, can we specify "text" somehow?

Reply:
The best I could do was "quote"